S23/tanvipatel23/CodereviewAssignment.java

```diff
@@ -0,0 +1,30 @@
+new Locale(language)
+new Locale(language, country)
+new Locale(language, country, variant)
+
+class JavaCharactertoLowerCaseExample1 {  
+  public static void main(String[] args) {  
+		
+		String s2 = Normalizer.normalize(s, Normalizer.Form.NFD);
+    		return s2.replaceAll("(?s)\\p{InCombiningDiacriticalMarks}", "");
+
+     char ch1, ch2, ch3, ch4;    
+      ch1 = 'TANVI';  

```

Comment:
hard code may not help if we want to use different word it would be great if we use flexible